pegasus/sites.v3/hourofcode.com/views/signup_form.haml

```diff
@@ -1,6 +1,7 @@
 - require 'country_codes'
 %script{type: "text/javascript", src: "/js/signup-form.js"}
 - signup_submit_error_message = hoc_s(:signup_submit_error_message).gsub(/"/,"\\\\\"")
+- signup_submit_census_error_message = hoc_s(:signup_submit_census_error_message)
```

Comment:
I think you'll need the `.gsub(/"/,"\\\\\"")` weirdness on this too. Because we're copying the translation string into your JavaScript string, a translation with a `"` won't be escaped.